packages/map-template/src/components/Search/Search.jsx

```diff
@@ -140,6 +140,15 @@ function Search({ onSetSize, isOpen }) {
         setSearchResults(locations);
         setFilteredLocations(locations);
         setShowNotFoundMessage(locations.length === 0);
+
+        // Handles updates to scroll buttons when the category changes.
+        // When a category changes, the scroll buttons need to have their enabled/disabled states updated.
+        // Since some categories might load before the SCSS element is fully rendered, we listen for the 'transitionend' event.
```

Comment:
"SCSS element"? 🤔 Did you mean DOM element?

Reply:
✅